### ISSUE TITLE
Problem with pick and Project operator.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1922]
An LLNL user was able to get VisIt to to fail when picking on a plot with the Project operator. VisIt produced an error message and pick did not work. When I tried parallel VisIt on 12 processors on rzalastor with 2.5.1, it worked fine. When I did the same thing with a serial engine, pick did not work and produced the message:

Warning: Pick encountered an internal error (could not find zone corresponding to pick point).
Please contact a VisIt developer

1. Open ~whitlocb/ScottWilks/hydropc14030.root
2. Pseudocolor plot of rbin_7
3. Add a Project operator
4. Set the Project operator atts so it uses "Z-Axis Cylindrical (x <= x, y <= r) for the projection type
5. Draw
6. Enter zone pick mode and click on the plot. You should get the error message.

Again, when I ran in parallel, I did not get the error message. The user reported that he's been doing this operation successfully for years and that maybe the 2.5.x series of VisIt is breaking it.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Brad Whitlock
Original creation: 08/02/2012 06:09 pm
Original update: 08/16/2012 06:08 pm
Ticket number: 1148
